client/coral-framework/services/i18n.js

```diff
@@ -6,15 +6,17 @@ import merge from 'lodash/merge';
 import esTA from '../../../node_modules/timeago.js/locales/es';
 import frTA from '../../../node_modules/timeago.js/locales/fr';
 import pt_BRTA from '../../../node_modules/timeago.js/locales/pt_BR';
+import da_DK from '../../../node_modules/timeago.js/locales/da_DK';
```

Comment:
The locales for timeago is called `da`. Also the whole path should be `timeago.js/locales/da` without `../../../node_modules/`. You change it for all others if you are already there :-)

Reply:
Is it `da` or `ta`?